packages/peregrine/lib/talons/MiniCart/useMiniCart.js

```diff
@@ -29,6 +29,7 @@ export const useMiniCart = () => {
         !((cartState.isEmpty && step === 'cart') || isLoading || isEditingItem);
 
     const isMiniCartMaskOpen = step === 'form';
+    const isOpen = drawer === 'cart';
```

Comment:
This was removed as a prop being passed by `App` since we already reference `useAppContext` here anyways. Might as well just fetch the state we need in `useMiniCart`.